Makefile

```diff
@@ -16,9 +16,20 @@
 # ****************************************************************************
 
 ifeq ($(BOLOS_SDK),)
-$(error Environment variable BOLOS_SDK is not set)
 endif
 
+ifneq ($(BOLOS_ENV),)
+$(info BOLOS_ENV=$(BOLOS_ENV))
+GCCPATH   := $(BOLOS_ENV)/gcc_nano_s_se200_and_nano_x_se124_compatible/bin/
+endif
+
+ifeq ($(TARGET),NANOX)
```

Comment:
You can compile on both S and X using the same clang, could you please remove this ?

Reply:
if you prefer, you can add a check to skip this part if `CLANGPATH_NANO_X` and `CLANGPATH_NANO_S` are empty, it would fix the issue for our CI

Reply:
Yes, please, add this check.

Reply:
Done